meta/main.yml

```diff
@@ -9,6 +9,13 @@ galaxy_info:
     versions:
     - 7
     - 8
+  - name: SLES
+    versions:
+    - 12
+    - 15
+  - name: opensuse
+    versions:
+    - all
```

Comment:
Lets be specific here that this was tested on OpenSuse 15. OpenSUSE 15.x is currently the only supported release. older are in the EOL (end-of-life). Is it OK to change this to `15` only instead of `all`?

Reply:
Sure.

Reply:
Fixed.